paddle/fluid/framework/ir/multi_devices_graph_pass/multi_devices_graph_pass.cc

```diff
@@ -1106,7 +1106,7 @@ static int MultiDevSSAGraphBuilderRegister(const std::string &builder_mode) {
       .RequirePassAttr(paddle::framework::details::kPlaces)               \
       .RequirePassAttr(paddle::framework::details::kLocalScopes)          \
       .RequirePassAttr(paddle::framework::ir::kStrategy)                  \
-      .RequirePassAttr(paddle::framework::ir::kNRanks)
```

Comment:
Because `kNRanks` can be used in other`pass`, and `kNRanks` is placed in `multi_device_graph_pass` before, now, it is moved to `multi_devices_helper.h`.